README.md

```diff
@@ -1,38 +1,50 @@
-jupyter-vim-binding
-========================
-Enable Vim key bindings in [jupyter](https://jupyter.org/) or [IPython notebook](http://ipython.org/notebook.html).
+#Jupyter VIM Binding
+
+This extension enables VIM keybindings in [Jupyter Notebook](https://jupyter.org/) (4.0+).
```

Comment:
Yeah, technically IPython notebook 4.0 use Jupyter thus what you pointed is completely correct.
But I'm afraid that people may get confused while people may think what they are using is "IPython notebook" and have no idea about "Jupyter" (like me, I confused when I update IPython notebook from 3 to 4). So I prefer to add a keyword "IPython notebook (4.0)" in case.